lib/fetcher.rb

```diff
@@ -18,32 +20,32 @@ def run_solr_query(params, method = 'select')
     response
   end
 
-  # Given the user's querystring parameters, and a fedora type, return a solr response containing all of the objects associated with that type
+  # Given the user's querystring parameters, and a object type, return a solr response containing all of the objects associated with that type
   # (potentially limited by rows or date if specified by the user)
   # @param params [Hash] querystring parameters from user, which could be an empty hash
-  # @param ftype [String] fedora object type, could be :collection
+  # @param ftype [String] object type, could be :collection
   # @return [Hash] solr response
-  # @example find_all_fedora_type(params,:collection)
-  def find_all_fedora_type(params, ftype)
+  # @example find_all_object_type(params,:collection)
+  def find_all_object_type(params, ftype)
     # ftype should be :collection (or other symbol if we added more since this was updated)
```

Comment:
might be good to ensure that `OBJECT_TYPES.include?(ftype)` or fail with `ArgumentError`, otherwise the Solr query will fail silently.